cognite/cli/cli.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import argparse
+import os
+import sys
+
+from cognite.cli.cli_models import CogniteMLCLI
+
+
+class CogniteCLI:
+    def __init__(self):
+        """cognite <service> [<args>]
+
+The available Cognite services are:
+    models      Model hosting
+"""
+        parser = argparse.ArgumentParser(description="Command line interface for CDP.", usage=self.__init__.__doc__)
+        parser.add_argument("service", help="Service to run", choices=["models"])
+        # parse_args defaults to [1:] for args, but you need to
+        # exclude the rest of the args too, or validation will fail
+        args = parser.parse_args(sys.argv[1:2])
+        if not os.getenv("COGNITE_API_KEY") or not os.getenv("COGNITE_PROJECT"):
+            print("You must set COGNITE_API_KEY and COGNITE_PROJECT environment variables.")
+            return
+        if not hasattr(self, args.service.replace("-", "_")):
+            print("Unrecognized service")
+            parser.print_help()
+            exit(1)
+        # use dispatch pattern to invoke method with same name
+        getattr(self, args.service)(sys.argv[2:])
+
+    def models(self, args):
+        """cognite models <command> [<args>]
+
+The available commands are:
+    source      Create a new model source package in the current directory.
+    go          Run tests, upload a source package and create a model with the same name.
```

Comment:
Maybe we come up with a more descriptive word?

Reply:
@NilsBarlaug-cognite